complaints/templates/complaints/my_complaints.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
What is this file for? You haven't incuded any data here.Remove thhis file if it is unnecessary.